package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar 28 10:19:44 WEST 2017 - knut.anderssen@suse.com
+
+- backported
+  - SlideShow: Escape plain text release notes being shown properly
+    in RichText (bsc#1028721).
+- 3.1.215
```

Comment:
I will merge now against CaaSP, in CaaSP current version is 3.1.214.1 so after the merge it could be 3.1.215.1 if not backported 3.1.214.1 is that correct?